### DISCUSSION TITLE
So you change your licence to "Copyright (C) 2019 Yoshifumi Fuyuno"

### DISCUSSION THREAD

**Discussion by githubbapoopa** [YoshifumiFuyuno/Search-from-Popup-or-ContextMenu#54] [Q&A]
So I see with 0.6.3 you changed your license to "Copyright (C) 2019 Yoshifumi Fuyuno"?

What licence ist that exactly? A fantasy licence not to be forced to release code by GPL?

Was it because everybody rightfully asked to release the code?

The code that - in the first place you took from SSS? With the best wishes from its developer even?

And now you decided to keep the code for yourself after people startet to show interest in your mod after so much recommendation?

Basically this is stealing. Or worse hiding...

Is that how code works for you?

Poor decision > Dead end project

All the best!

https://github.com/CanisLupus/swift-selection-search/issues/244#issuecomment-1061129489

https://github.com/YoshifumiFuyuno/Search-from-Popup-or-ContextMenu/discussions/52
